core/src/assets/defaultConfig.json

```diff
@@ -60,6 +60,26 @@
           ]
         }
       },
+      "DISABLED_NODES": {
+        "selectors": []
+      },
+      "HIDDEN_NAMESPACES": {
```

Comment:
every feature should have `isEnabled` field as well.

Reply:
I agree